src/KafkaFlow.OpenTelemetry/ActivitySourceAccessor.cs

```diff
@@ -2,40 +2,34 @@
 
 namespace KafkaFlow.OpenTelemetry
 {
-    using System;
-    using System.Collections.Generic;
     using System.Diagnostics;
     using System.Reflection;
     using Conventions = SemanticConventions::OpenTelemetry.Trace.TraceSemanticConventions;
 
-    internal static class ActivitySourceAccessor
+    /// <summary>
+    /// Properties that sets up KafkaFlow ActivitySource
+    /// </summary>
+    public static class ActivitySourceAccessor
     {
         internal const string ActivityString = "otel_activity";
-        internal const string ExceptionEventKey = "exception";
         internal const string MessagingSystemId = "kafka";
         internal const string AttributeMessagingOperation = "messaging.operation";
         internal const string AttributeMessagingKafkaMessageKey = "messaging.kafka.message.key";
         internal const string AttributeMessagingKafkaMessageOffset = "messaging.kafka.message.offset";
         internal static readonly AssemblyName AssemblyName = typeof(ActivitySourceAccessor).Assembly.GetName();
-        internal static readonly string ActivitySourceName = AssemblyName.Name;
         internal static readonly string Version = Assembly.GetExecutingAssembly().GetName().Version.ToString();
-        internal static readonly ActivitySource ActivitySource = new(ActivitySourceName, Version);
 
-        public static void SetGenericTags(Activity activity)
-        {
-            activity?.SetTag(Conventions.AttributeMessagingSystem, MessagingSystemId);
-        }
+        /// <summary>
+        /// ActivitySource name to be used when adding
+        /// KafkaFlow as source to an OTEL listener
+        /// </summary>
+        public static readonly string KafkaFlowActivitySourceName = AssemblyName.Name;
```

Comment:
What do you think about extracting this to a different file?
Something like:

```csharp
public static KafkaFlowInstrumentation{
public static readonly string ActivitySourceName = AssemblyName.Name;
}
```

Reply:
Yeah, `KafkaFlowInstrumentation.ActivitySourceName` makes more sense than `ActivitySourceHelper.KafkaFlowActivitySourceName`

Can you check the commit and see if you like it?
https://github.com/Farfetch/kafkaflow/pull/476/commits/d77bf68c7067975966ea70174050c71af61cfa94

Reply:
Sounds good to me